src/core/CCList.mli

```diff
@@ -302,18 +308,18 @@ val sublists_of_len :
     @since 1.5 with labels *)
 
 val intersperse : 'a -> 'a list -> 'a list
-(** Insert the first argument between every element of the list.
+(** [intersperse x l] inserts the element [x] between adjacent elements of the list [l].
     @since 2.1, but only
     @since 2.2 with labels *)
 
 val interleave : 'a list -> 'a list -> 'a list
-(** [interleave [x1…xn] [y1…ym]] is [x1,y1,x2,y2,…] and finishes with
+(** [interleave [x1…xn] [y1…ym]] is [x1;y1;x2;y2;…] and finishes with
```

Comment:
The `[` and `]` will not be print in the documentation for `[x1;y1;x2;y2;…]`, if you want to have the OCaml representation of a list you should add an extra `[]` around it.

Reply:
OK.